SwiftWisdom/Core/UIKit/UIView+Geometry.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  UIView+Geometry.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    public var ip_width: CGFloat {
+        return CGRectGetWidth(frame)
```

Comment:
I'm perfectly fine leaving them in. I do think however, that once you have ip_width and ip_frameWidth, it gets quite close to view.bounds.width and view.frame.width. I think we can always add them, since one can always choose not to use them!